modules/Store.js

```diff
@@ -0,0 +1,31 @@
+export default class Store {
+  static getLists = () => {
+    let Lists;
+    if (localStorage.getItem('Lists') === null) {
+      Lists = [];
+    } else {
+      Lists = JSON.parse(localStorage.getItem('Lists'));
```

Comment:
- Please follow the `DRY` rule. As you can see in the above code-snippet you are repeating yourself for getting `LocalStorage` data which will be problematic for you later on to maintain the code.
- I would like to suggest you, kindly change using `variable` for storing the `localStorage` data and use that variable to perform conditional rendering.